server/errors.go

```diff
@@ -21,7 +21,7 @@ func (me MetaError) Error() string {
 	return fmt.Sprintf("Error: %s (Mode: %s, CertVersion: %b)",
 		me.Err.Error(),
 		me.Meta.Mode,
-		me.Meta.CertVersion)
+		me.Meta.Version)
```

Comment:
why change the name? Its unclear what Version refers to here. I still very much hate `meta`. Can we refactor that? Perhaps in a subsequent PR?

Reply:
This name change was part of the change set as I picked it up.

I will fix it as part of [this issue](https://linear.app/eigenlabs/issue/DAINT-325/standardize-version-related-language-in-proxy)